app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java

```diff
@@ -50,6 +50,7 @@
 
     private View rootView;
 
+    //TODO: What is this doing? What is special about these entries?
     public static String options = "Available|In use|Almost done|End of cycle";
```

Comment:
This set of options is scattered around the code. It should be collected into one place. 

What is it doing @hughesjeff ? Did you write these parts?